src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/dllmain.cpp

```diff
@@ -63,6 +63,8 @@ BOOL WINAPI DllMain(HMODULE hModule,
     return TRUE;
 }
 
+DWORD dwIISServerVersion;
```

Comment:
If we're going to go the global route, we should at least put it at the top of the file with the other globals.